### ISSUE TITLE
Drom 0.4 is tagged but 0.3 is still the current version on opam

### ISSUE THREAD

**Issue by crispywalrus** [OCamlPro/drom#149]
I wanted to try it out, and it seems like it would be a nice tool. However, the fact that it seems to have missed publishing your 0.4 release isn't the greatest advertisement for its functionality. I can of course pin the version, and I have, but it would be nice to have the current stable release as the opam default

**Comment by lefessan** [comment]
Yes, we need to push a new version in `opam` ! It will be 0.5.0 because 0.4.0 has been in dev too long.
